File: 0125-valid-palindrome/0125-valid-palindrome.java
class Solution {
    public boolean isPalindrome(String s) {
      s=s.replaceAll("[^a-zA-Z0-9]","");
      s=s.toLowerCase();
      System.out.println(s);
      boolean f=true;
      int i;
      for(i=0;i<s.length()/2;i++){
        if(s.charAt(i)!=s.charAt(s.length()-i-1))
           {f=false;
           break;}
      }
      return f;
    }
}

File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution(object):
    def isPalindrome(self, s):
        st = ""

        for ch in s:
            if ch.isdigit() or ch.isalpha():
                st += ch.lower()

        left = 0
        right = len(st) - 1

        while left < right:
            if st[left] != st[right]:
                return False
            left += 1
            right -= 1

        return True